### PR TITLE
Hyperweb embed

### LINKED ISSUE #99: Configure hyperweb content for the dashboard
EIC has received an urgent request to stage content that appears on the hyperwall at NASA HQ on the front page of [earth.gov](http://earth.gov/) ASAP. 

@agurvich has put together a small web application that does this at https://svs.gsfc.nasa.gov/webapps/hyperweb and thinks we can replace the carousel on https://earth.gov/stories/hyperwall with an embedded version of this hyperweb and then adding a banner like the one for the GHG Center that points to the hyperwall page.

Where did we land on about the embed carousel and embedding content without the border?

**Acceptance Criteria**
- [ ] Add hyperwall content to dashboard by replacing carousel

### PR DESCRIPTION
**Pull request by agurvich**
## What am I changing and why

Adds banner to home page and embeds hyperweb content in the hyperwall page, resolves #99 

## How to test
local-build

Unfortunately, due to CSP issues the hyperweb is not going to show up in the netlify preview. I asked the SVS sysadmin to whitelist all of *.netlify.app but he hasn't gotten back to me yet. For my local development, I am using a reverse proxy I setup that adds localhost:* to frame-ancestors so that I can frame these. Here's a screenshot of what that looks like. Even still I'm getting 403'd on a lot of of the SVS content. He did add earth.gov to the frame-ancestors policy so it _should_ work in production. 

## ⚠️ Checks

- [ ] I have confirmed that [updating the `veda-ui` submodule](https://github.com/NASA-IMPACT/veda-config-ghg/blob/main/docs/DEVELOPMENT.md#development) is needed and **only done so** if that's the case.

### CODE REVIEW

File: overrides/components/hyperwall-banner.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    <Fold>
+      <FoldBody>
+        <BottomContent width={width}>
+          <InfoImageContent>
+            <div>
+              <StyledVarHeading size="small" as="h2">
+                Live Earth Dashboard
+              </StyledVarHeading>
+              <span>
+                An interactive way to explore what's happening on Earth right now.
```
Comment (slesaad) [inline overrides/components/hyperwall-banner.tsx:97]: I'd remove the period at the end to be consistent with the GHG Center banner

File: overrides/components/hyperwall-banner.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+              >
+                Visit the Live Earth Dashboard
+              </Button>
+            </div>
+          </InfoImageContent>
+        </BottomContent>
+      </FoldBody>
+    </Fold>
+  )
+};
```
Comment (slesaad) [inline overrides/components/hyperwall-banner.tsx:114]: nit: newline

**Comment by slesaad** [comment]
Looks good to me, but I'll let @hanbyul-here or @sandrahoang686 chime in

File: overrides/components/hyperwall-banner.tsx (hunk 2 above)
Comment (agurvich) [reply overrides/components/hyperwall-banner.tsx:114]: Between the `)` and the `}`? 

I inherited that from copying the GHG Center's banner: 

https://github.com/NASA-IMPACT/veda-config-eic/blob/develop/overrides/components/visit-ghg.tsx#L95

File: overrides/components/hyperwall-banner.tsx (hunk 1 above)
Comment (agurvich) [reply overrides/components/hyperwall-banner.tsx:97]: Good call

File: overrides/components/hyperwall-banner.tsx (hunk 2 above)
Comment (slesaad) [reply overrides/components/hyperwall-banner.tsx:114]: at the end of the file. GitHub complains if it's not present
image

I had been following this convention like a sheep without knowing why, but I finally [learned](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795) to reply to this comment 😅

**Comment by sandrahoang686** [comment]
code lgtm as well

**Comment by hanbyul-here** [comment]
My nitpick is 'Banner' is the term for UI that is supposed to be at the top of the website afaik (ex. https://m2.material.io/components/banners#usage). If you @agurvich are planning to change the code anyway, it might be worth naming `HyperwallBanner` to something like `HyperwallCTA`. But otherwise, :shipit:

**Comment by agurvich** [comment]
Easy enough to change now rather than something to clean up in the future

**Review by hanbyul-here — APPROVED** [review]

**Comment by agurvich** [comment]
Cheers everyone! Let's cross our fingers and hope it looks good on staging.earth.gov haha

**Pull request merged**